components/progressbar/index.css

```diff
@@ -235,7 +235,7 @@
 	}
 
 	.spectrum-ProgressBar-track {
-		background-color: var(--spectrum-progressbar-track-color-white);
+		background: var(--mod-progressbar-track-color-white, var(--spectrum-progressbar-track-color-white));
```

Comment:
I realize now that maybe this mod isn't necessary. If it's for `staticWhite`, maybe we purposely didn't allow for a mod property. Should I remove this? It was just the only one here that _didn't_ have a mod, so I added thinking I was being consistent, but maybe the point was to keep the track a single, defined color.

Reply:
Correct, yes staticWhite is an edge case we don't want to surface too much or make too customizable.